tests/scripts/thread-cert/message.py

```diff
@@ -166,6 +174,22 @@ def assertMleMessageContainsTlv(self, tlv_class_type):
                 break
 
         assert(contains_tlv == True)
+        return tlv
+
+    def assertAssignedRouterQuantity(self, router_quantity):
+        """ To confirm if Leader contains the Route64 TLV with router_quantity assigned Router IDs.
+
+        Args:
+            router_quantity: the quantity of router
```

Comment:
nit: add `. `in the end
and also other comments in this PR.

Reply:
fixed